packages/layout/src/components/cards/employer/employer.tsx

```diff
@@ -93,12 +94,7 @@ const EmployerSubtitle: React.FC<Partial<Employer>> = ({
 		[statutoryEmployer],
 	);
 
-	return (
-		<>
-			<P className={styles.noMarginBottom}>{title}</P>
-			<P className={styles.noMarginBottom}>{subtitle}</P>
-		</>
-	);
+	return <Subtitle main={title} secondary={subtitle} mainNotBold={true} />;
```

Comment:
It's usually easier to understand if you use positive property names, ie use `mainBold={false}` rather than `mainNotBold={true}`.

Reply:
prop renamed to `mainBold`